src/util.rs

```diff
@@ -433,6 +427,27 @@ pub fn build_rs() -> TokenStream {
     }
 }
 
+pub trait Suffix {
+    fn replace_suffix(&self, suffix: &str) -> Cow<str>;
+    fn clean_suffix(&self) -> Cow<str> {
+        self.replace_suffix("")
+    }
+}
```

Comment:
would it maybe make more sense to call these `expand_dim`, `remove_dim`?

Reply:
Ok. Done.